frontend/assets/tailwind.css

```diff
@@ -3,14 +3,15 @@
 @tailwind utilities;
 
 :root {
-    --white: #fff;
+    ---background-light: #cad2d369;
+    --Primary-light-color: #eaeaff;
```

Comment:
Why sometimes is uppercase and sometimes lowercase?

Reply:
Colors are already removed from tailwind.css